sample/sample.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"crypto/md5"
+	"fmt"
+	"os"
+)
+
+func main2() {
+	for _, arg := range os.Args {
+		fmt.Printf("%x - %s\n", md5.Sum([]byte(arg)), arg)
```

Comment:
**[gosec]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
[CWE-326] Use of weak cryptographic primitive (Rule:G401, Severity:MEDIUM, Confidence:HIGH)